### ISSUE TITLE
Have bot collect standups

### ISSUE THREAD

**Issue by arinehouse** [tburnam/PMBot#2]
Standups should be flagged first (i.e. `!standup` or `#standup`) and cause the bot to start a conversation in three steps/questions:
- What did you accomplish last week?
- What do you plan to accomplish this week?
- Any blockers, questions, or concerns?

**Comment by tburnam** [comment]
Should this be in project channels or in direct messages to the PM bot? I think in the project channel would cause clutter. Maybe the PM can trigger the stand up and the bot will direct message all members. Thoughts?

**Comment by arinehouse** [comment]
Agreed, should be direct message. Probably best to have it set by the PM on a weekly schedule and then have it direct message everyone on the team.
